src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -3,6 +3,7 @@
 
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Default parameter set is 'Ping'
```powershell
Expected exactly '::1', but got 127.0.0.1.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 37
37:             $result.Address | Should -BeExactly $targetAddressIPv6
```
Test-Connection.Ping.Allows us to Force IPv6
```powershell
Expected exactly '::1', but got $null.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 114
114:             $result.Address | Should -BeExactly $targetAddressIPv6
```
Test-Connection.Ping.can convert IPv6 addresses to IPv4 when required
```powershell
Value cannot be null. (Parameter 'address')
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 119
119:             $result = Test-Connection $targetAddressIPv6 -IPv4 -Count 1

```
Test-Connection.Ping.can convert IPv4 addresses to IPv6 with -IPv6
```powershell
Value cannot be null. (Parameter 'address')
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 124
124:             $result = Test-Connection $targetAddress -IPv6 -Count 1

```